File: PROGRAMMERS/lv0/숫자 비교하기.js
function solution(num1, num2) {
  return num1 === num2 ? 1 : -1;
}

let inps = [
  [2, 3],
  [11, 11],
  [7, 99],
];

for (let i of inps) {
  console.log(solution(i[0], i[1]));
}

File: PROGRAMMERS/lv0/python/숫자 비교하기.py
def solution(num1, num2):
    return 1 if num1==num2 else -1

inps = [
    [2,	3],
    [11, 11],
    [7,	99]
]

for (n1, n2) in inps:
    print(solution(n1, n2))